.github/dependabot.yml

```diff
@@ -9,3 +9,504 @@ updates:
   directory: "/"
   schedule:
     interval: daily
+
+
+  - package-ecosystem: maven
+    directory: /m2e-maven-runtime/org.eclipse.m2e.archetype.common
+    schedule:
+      interval: daily
+
+  - package-ecosystem: maven
+    directory: /m2e-maven-runtime/org.eclipse.m2e.maven.runtime
+    schedule:
+      interval: daily
+
+  - package-ecosystem: maven
```

Comment:
Configuring each Project seems to be cumbersome, isn't the configuration for the project root inherited?

Reply:
Hi @HannesWell , as far I am aware config file doesn't support wildcards which help to manage this configuration in easier way. Sorry about that
Here you will find more information https://github.com/dependabot/dependabot-core/issues/2178